### PR TITLE
ACFT HF NLP PuP v0.0.1 pipeline components

### PR DESCRIPTION
**Pull request by skanakamedal**

### CODE REVIEW

File: training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml [hunk 1]
```diff
@@ -523,7 +523,7 @@ jobs:
       mlflow_model: ${{parent.jobs.translation_finetune.outputs.mlflow_model_folder}}
   compute_metrics:
     type: command
-    component: azureml://registries/azureml-preview/components/compute_metrics/versions/0.0.9.preview
+    component: azureml://registries/azureml/components/compute_metrics/versions/0.0.1
```
Comment (vizhur) [inline training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml:526]: can we start with v1 in a public registry?

Comment (skanakamedal) [reply training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml:526]: @vizhur you mean version to be set to 1.0.0? We already published command components with 0.0.1 and hence maintaining the same version.

Comment (skanakamedal) [reply training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml:526]: cc: @imbalu007

**Review by ankushbhatia2 — APPROVED** [review]

File: training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml (hunk 1 above)
Comment (imbalu007) [reply training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml:526]: @vizhur, the components are still in public preview. At GA we plan to go with v1

Comment (vizhur) [reply training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml:526]: sure, 0.0.1 doesn't look good for PuP either

**Review by vizhur — APPROVED** [review]

**Pull request merged**